Gruntfile.js

```diff
@@ -46,59 +46,26 @@ module.exports = function(grunt) {
       }
     },
 
-    styleguide: {
-            dist: {
-                options: {
-                    name: 'eRegs Styleguide'
-                },
-                files: {
-                    '<%= env.frontEndPath %>/docs/styleguide': '<%= env.frontEndPath %>/css/'
-                }
-            }
-        },
-
     /**
-     * JSHint: https://github.com/gruntjs/grunt-contrib-jshint
+     * ESLint: https://github.com/sindresorhus/grunt-eslint
```

Comment:
That is interesting. I honestly didn't look long, I just saw that this plugin existed and is by a well known community member.